git-lfs.yaml

```diff
@@ -22,18 +22,22 @@ environment:
       - go
       - git
 pipeline:
-  - uses: fetch
+  - uses: git-checkout
     with:
-      uri: https://github.com/git-lfs/git-lfs/archive/v${{package.version}}.tar.gz
-      expected-sha256: d5eeb9ee33188d3dd6a391f8a39b96d271f10295129789e5b3a1ac0e9f5114f5
+      repository: https://github.com/git-lfs/git-lfs
+      tag: v${{package.version}}
+      expected-commit: 77deabdf9a18f8a07ecfd3e0f4aa572ffbbab8d4
+      destination: git-lfs
+
   - runs: |
+      cd git-lfs
       # mitigate CVE-2021-38561
       go get golang.org/x/text
 
       # mitigate CVE-2022-27191
       go get golang.org/x/crypto/ssh
   - runs: |
-      make EXTRA_GO_FLAGS="-v -tags netcgo" GIT_LFS_SHA="v${{package.version}}" VERSION="v${{package.version}}"
-  - runs: |
+      cd git-lfs
```

Comment:
It would be nice if git-checkout `cd`ed into there for us... 🤔

Reply:
i think whenever we create a new step with `runs`, it will be change back to workspace root?

Reply:
Yeah, new pipeline steps start at whatever `working-directory` is, which is by default the workspace root.  The packaging I did for .NET 6 shows how you can leverage this in interesting ways: https://gist.github.com/kaniini/0e7e90f23dc04b6db7d3cd30a030f1f7